lib/samples/add_feature_layers/add_feature_layers_sample.dart

```diff
@@ -30,28 +30,46 @@ class AddFeatureLayersSample extends StatefulWidget {
 }
 
 class _AddFeatureLayersSampleState extends State<AddFeatureLayersSample> {
+  // create a map view conroller.
   final _mapViewController = ArcGISMapView.createController();
+  // create a map with a topographic basemap style.
   final _map = ArcGISMap.withBasemapStyle(BasemapStyle.arcGISTopographic);
+  // create a list of feature layer sources.
   final _featureLayerSources =
       List<DropdownMenuItem<String>>.empty(growable: true);
+  // create a variable to store the selected feature layer source.
   String? _selectedFeatureLayerSource;
```

Comment:
We should use an enum for the underlying values rather than a string. We did that in show_service_area_sample.dart, with `enum Selection`. The benefit of an enum is that it enforces a proper value, whereas a String can be anything. Making this change would require changing a couple different places that specify `<String>`.